bin/src/command_line.dart

```diff
@@ -1,17 +1,102 @@
 library cmdpart;
 
 import 'dart:async';
+import 'dart:io' show Directory, File, FileSystemEntity, Platform;
 import "dart:math" show max;
 
 import "package:args/args.dart";
+import 'package:dart_coveralls/dart_coveralls.dart' show CommandLineClient, log;
+import 'package:logging/logging.dart';
+import 'package:stack_trace/src/chain.dart';
 
 export "package:args/args.dart";
 
+final Logger _log = new Logger("dart_coveralls");
+
 abstract class CommandLinePart {
   final ArgParser parser;
 
   CommandLinePart(this.parser);
 
+  bool handleLogging(ArgResults res) {
+    var logLevelStr = res['log-level'];
+
+    Level logLevel = Level.LEVELS
+        .firstWhere((level) => level.name.toLowerCase() == logLevelStr);
```

Comment:
This will cause a very strange termination when you have an invalid arg here. :)  But OK.  I'm slightly surprised package:logging doesn't already provide some tooling around handling args.